ntbs-service/wwwroot/css/header.scss

```diff
@@ -18,13 +18,59 @@
   } 
 }
 
-
 .header-navigation-list {
     margin-bottom: 0;
 
     .header-navigation-list-item {
         font-size: large;
     }
+
+    ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+    }
+  
+    ul li {
+        margin: 0;
+        padding: 5px;
+    }
+  
+    li.dropdown-menu {
+        display: inline-block;
+    }

```

Comment:
A bit surprised we're styling directly on html elements here - any reason why classes aren't useful here?

Reply:
I was styling html elements of particular parent, but can user classes instead